src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -585,6 +585,9 @@ public bool TryGetValue(string key, out object? value)
     [UnconditionalSuppressMessage("Trimming", "IL2026",
         Justification = "The constructor that would result in _propertyStorage being non-null is annotated with RequiresUnreferencedCodeAttribute. " +
         "We do not need to additionally produce an error in this method since it is shared by trimmer friendly code paths.")]
+    [UnconditionalSuppressMessage("AOT", "IL3050",
+        Justification = "The constructor that would result in _propertyStorage being non-null is annotated with RequiresDynamicCodeAttribute. " +
+        "We do not need to additionally produce an error in this method since it is shared by AOT friendly code paths.")]
```

Comment:
These suppressions are no longer valid because the `public RouteValueDictionary(object? values)` constructor is no longer annotated with `RequiresDynamicCodeAttribute`.

Reply:
Some dynamic code was still used in `RouteValueDictionary(object? values)`, but I replaced it with some simple reflection is dynamic code isn't supported.